doc/api/util.md

```diff
@@ -865,6 +865,96 @@ added: v8.0.0
 * {symbol} that can be used to declare custom promisified variants of functions,
 see [Custom promisified functions][].
 
+## util.report.getReport([err])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `err` {Object}
+* Returns: {Object} Returns the diagnostics report as an Object
+
+Generates a JSON-formatted diagnostic report summary of the running process.
+The report includes JavaScript and native stack traces, heap statistics,
+platform information, resource usage etc.
+
+```js
+const { report } = require('util');
+const data = report.getReport();
+console.log(data);
+```
+
+Additional documentation on diagnostic report is available
+at [report documentation][]
+
+## util.report.setDiagnosticReportOptions([options]);
+<!-- YAML
+added: REPLACEME
+-->
+
+Set the runtime configuration of diagnostic report data capture. Upon invocation
+of this function, the runtime is reconfigured to generate report based on
+the new input.
+
+* `options` {Object}
+  * `events` {string[]}
+    * `signal`: generate a report in response to a signal raised on the process.
+    * `exception`: generate a report on unhandled exceptions.
+    * `fatalerror`: generate a report on internal fault.
+      (such as out of memory errors or native assertions)
+  * `signal` {string} sets or resets the signal for report generation
+    (not supported on windows). Default signal is `SIGUSR2`.
+  * `filename` {string} name of the file to which the report will be written.
+  * `path` {string} location at which the report will be generated.
```

Comment:
Is it the directory? From reading the code I think it is, I think `directory` would be less confusing..

Reply:
fixed, changed from `location` to `directory`.